miner/block_builder.go

```diff
@@ -194,9 +195,17 @@ func (t *BlockBuilder) createBlock(id types.LayerID, atxID types.AtxId, eligibil
 				log.Uint64("bottom", uint64(bottom)), log.Uint64("top", uint64(top)),
 				log.Uint64("hdist", uint64(t.hdist)), log.Err(err))
 		}
```

Comment:
i think we should break this down and do a getVotesForBlock or something to contain all this vote logic

Reply:
Done